appinfo/info.xml

```diff
@@ -7,6 +7,15 @@
     <author>Hendrik Leppelsack, Thomas Müller, Alexander Weidinger, Jan-Christoph Borchardt, Tom Needham</author>
     <version>1.3.1.0</version>
     <category>other</category>
+    <documentation>
+    	<user>https://doc.owncloud.org/server/9.1/user_manual/pim/contacts.html</user>
+    	<admin>https://doc.owncloud.org/server/9.1/admin_manual/configuration_server/occ_command.html?highlight=contact#dav-commands</admin>
+    	<developer>https://doc.owncloud.org/missing</developer>
+    </documentation>
+    <category>tool</category>
+    <website>https://owncloud.org</website>
+    <bugs>https://github.com/owncloud/theapp/issues</bugs>
+    <repository type="git">https://github.com/owncloud/theapp.git</repository>
```

Comment:
Sorry, apparently I was distracted and a bit early to commit, not having 
replaced the default information.

Fixed this by replacing the theapp links, and put the README.md as 
dev-documentation.

missing is more a link with a ;-). But it could be used as an invitation 
page to participate by creating this information as well...


Am 2016-06-15 um 20:48 schrieb Thomas Müller:
>
> In appinfo/info.xml 
> <https://github.com/owncloud/contacts/pull/428#discussion_r67222460>:
>
> > @@ -7,6 +7,15 @@
> >      <author>Hendrik Leppelsack, Thomas Müller, Alexander Weidinger, Jan-Christoph Borchardt, Tom Needham</author>
> >      <version>1.3.1.0</version>
> >      <category>other</category>
> > +    <documentation>
> > +    	<user>https://doc.owncloud.org/server/9.1/user_manual/pim/contacts.html</user>
> > +    	<admin>https://doc.owncloud.org/server/9.1/admin_manual/configuration_server/occ_command.html?highlight=contact#dav-commands</admin>
> > +    	<developer>https://doc.owncloud.org/missing</developer>
> > +    </documentation>
> > +    <category>tool</category>
> > +    <website>https://owncloud.org</website>
> > +    <bugs>https://github.com/owncloud/theapp/issues</bugs>
> > +    <repository type="git">https://github.com/owncloud/theapp.git</repository>
>
> theapp ❓
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/owncloud/contacts/pull/428/files/8660907fcde80551acfdf4241f3dd40b6ed9cb88#r67222460>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ALEnw8tdwpnATwvAAzDcHsNXdoS5TZAhks5qMEj9gaJpZM4Is7TF>.
>

-- 
Jan Klostermann
- The Agile Management Sounding-Board -
Systems Engineer & Mediator
Am Isarkanal 24/15
81379 München

Tel.: +49-89/74563382
Mobil: +49-179/4628819
Sip: +49-89/21551663 <sip:08921551663@sipgate.de>
Skype: jan.klostermann <callto://jan.klostermann/>
Threema: P58VSJUY <threema://add?id=P58VSJUY>
Email: jk@janklostermann.de <mailto:jk@janklostermann.de>
Netzwerke: Xing <https://www.xing.com/profile/Jan_Klostermann>, LinkedIn 
<http://de.linkedin.com/in/janklostermann>